test/test.js

```diff
@@ -0,0 +1,49 @@
+var createS3UploadParams = require('../index').createS3UploadParams
+var reverse = require('../index').reverse
+var assert = require('assert');
+
+describe('Reverse', function () {
+    const tests = [
+        { word: 'abc', expected: 'cba' }
+    ];
+
+    tests.forEach(({ word, expected }) => {
+        it(`reverses ${word} correctly`, function () {
+            assert.equal(reverse(word), expected);
+        });
+    });
+});
+
+describe('CreateS3UploadParams', function () {
+    ip_ranges_example = {
+        "syncToken": "0123456789",
```

Comment:
Codacy has a fix for the issue: [Replace `····"syncToken"` with `syncToken`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
    syncToken: "0123456789",
```